### PR TITLE
Verify parts

### PR DESCRIPTION
**Pull request by picasdan9**

### CODE REVIEW

File: src/pages/VerifyParts.jsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            pageSize: 10
+          }}
+        />
+      </div>
+      <Dialog
+        open={open}
+        onClose={() => setOpen(false)}
+        maxWidth="md"
+        aria-labelledby="form-dialog-title"
+      >
```
Comment (elenacyl) [inline src/pages/VerifyParts.jsx:110]: - [ ] Add a 'cross' button on the top left or top right to close the popup
- [ ] Don't allow popup to close when we click background

File: src/pages/VerifyParts.jsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        <EditItem
+          onClose={() => setOpen(false)}
+          isDialog={true}
+          itemID={itemID}
+        />
```
Comment (elenacyl) [inline src/pages/VerifyParts.jsx:115]: Use different buttons for this form:
- [ ] Add a "cancel" button (on the left) - make it grey
- [ ] Replace confirm changes with "verify part" (on the right) - make it green
- [ ] Both buttons to be side by side
- [ ] Remove "delete item" option

Comment (elenacyl) [reply src/pages/VerifyParts.jsx:115]: Make sure that the only the buttons in this popup show as such. The other pages (e.g. edit item) should still have their original buttons.

Comment (picasdan9) [reply src/pages/VerifyParts.jsx:115]: @elenacyl does the button "verify part" button automatically changes the status to verifying, or it only submits whatever input is in the form (where the status might not be changed to verifying yet)?

Comment (elenacyl) [reply src/pages/VerifyParts.jsx:115]: The "verify part" button will automatically change the status to verifying and also submit whatever input is in the form. It should happen at the same time.

**Pull request merged**